README.md

```diff
@@ -193,124 +200,167 @@ Other acceptable location examples:
-- The tests scale to the amount of available CPU cores. Our code is very lightweight and shouldn't use too many of your resources, so in most cases we recommend running our probe as is. But if you're worried you can use `--cpuset-cpus="0-2"` to limit the number of available cores.
-- We rate-limit all users on the API level to avoid the abuse of network
-- No local network tests are allowed, only public endpoints.
+#### Probes share no UUIDs
+Our probes don't expose unique IDs that you can use to target them explicitly. Our probes don't expose unique IDs that you can use to target them explicitly. Instead, we recommend fine-tuning the probe selection by using and combining filters or specifying IDs from previous measurements, as described in the best practice section above.
 
-Read more about the [Globalping Probe](https://github.com/jsdelivr/globalping-probe) in the dedicated repo.
+This ensures that popular locations, like `from Amsterdam` or `from AWS`, are automatically load-balanced across multiple probes within the same location, preventing the overload of specific probes.
 
+#### Test result variability
+When requesting a specific number of probes, there is no guarantee that the API will respond with the exact amount.
 
-## Limits | WIP
+#### Probe availability rules
+Globalping exposes and lets you interact with probes that are currently **online**. 
 
-Our platform has multiple limits to avoid abusive behaviour and at the same time motivate people to contribute to the sustainability of our platform.
+Here's when a probe is considered online and available through our API:
+- **It's up-to-date.** A probe running an older version will be forced to auto-update before it becomes available.
+- **It passes quality control tests.** Every probe must pass a test, ensuring it's connected to a reasonably stable network without packet loss. This test is done regularly; if a probe fails, it gets taken offline.
 
-#### Global limits
+This whole process is completely automated and managed by our platform.
 
-These limits are applied per IP address regardless if an API key supplied or not.
+## Join the network – Run a virtual probe
+Globalping relies on the community to expand its probe network. While we maintain our own probes in key locations, we welcome any support from both corporate partners and individuals. 
+[Join our network](https://github.com/jsdelivr/globalping-probe) and help make the internet faster for everyone by running a virtual probe (or several).
 
-- 100 POST requests per minute per IP. No GET limits are implemented to support "real-time" use-cases.
-- A single measurement is limited to 200 probes per location and 500 total probes
+### Quick start guide
+You can run a Globalping probe on any machine that can run a Docker container, supporting x86 and ARM architectures. For example, you can use a rented VPS, a dedicated server with spare capacity, or even your locally hosted Raspberry PI.
```

Comment:
I think you forgot our only requirement "The only requirement is having an internet accessible device able to run docker containers."